packages/mikro-orm/src/index.ts

```diff
@@ -1,9 +1,14 @@
-import type { Options } from "@mikro-orm/core"
+import type {
+  AnyEntity,
+  EntityClass,
+  EntitySchema,
+  Options,
+} from "@mikro-orm/core"
 import { MikroORM, wrap } from "@mikro-orm/core"
 import * as defaultEntities from "./entities"
 
 import type { Adapter } from "next-auth/adapters"
-import { isPromise } from "util/types"
+import type { EntityClassGroup } from "@mikro-orm/core/dist/typings"
```

Comment:
would be good not to use deep imports - this will break with stable v5, as stable releases do not have the `dist` folder (only its contents are published for stable releases)

(will be happy to accept PR exposing this type)

Reply:
somehow this slipped through. have to on my list now!